### PR TITLE
[2/4] CUDNNv8 ResNet Fusion:  Add fused_scale_bias_add_relu OP

### PR DESCRIPTION
**Pull request by Tom-Zheng**
### PR types

New features
### PR changes

OPs
### Description

This PR adds fused_scale_bias_add_relu op, which is part of the CUDNNv8 ResNet fusion integration.

### CODE REVIEW

**Comment by Tom-Zheng** [comment]
@Xreki This PR is similar to https://github.com/PaddlePaddle/Paddle/pull/55026, would you please take a look?

**Comment by Tom-Zheng** [comment]
`PR-CI-OP-benchmark` and `PR-CI-Coverage` need manual approval and should not block review process.

**Review by zyfncg — APPROVED** [review]

**Comment by Tom-Zheng** [comment]
@phlrain would you please approve `check_dygraph=False`? This OP is only intended for static graph IR pass.
@luotao1 would you please approve use of `skip_check_grad_ci`; use of `skipIf `; setting `atol, rtol`; use of `const_cast`?

**Review by luotao1 — APPROVED** [review]

File: paddle/phi/api/yaml/fused_ops.yaml [hunk 1]
```diff
@@ -206,14 +206,24 @@
   backward: fused_rotary_position_embedding_grad
   support_dygraph_mode : true
 
-- op : fused_scale_bias_relu_conv_bnstats
+- op : fused_scale_bias_add_relu
+  args : (Tensor x1, Tensor scale1, Tensor bias1, Tensor x2, Tensor scale2, Tensor bias2, bool fuse_dual, bool exhaustive_search)
+  optional : scale2, bias2
+  output : Tensor(y)
```
Comment (yuanlehome) [inline paddle/phi/api/yaml/fused_ops.yaml:212]: 一般来说，单输出都取名为out

File: paddle/phi/infermeta/fusion.cc [hunk 2]
```diff
@@ -1821,6 +1820,32 @@ void FusedScaleBiasReluConvBnstatsInferMeta(
...
+                   phi::errors::InvalidArgument(
+                       "Argument scale2 and bias2 should be provided when "
+                       "fuse_dual is set, but got has_scale2=%d, has_bias2=%d, "
+                       "fuse_dual=%d.",
+                       has_scale2,
+                       has_bias2,
+                       fuse_dual));
+  }
+  // set output dims
+  y->set_dims(x1.dims());
```
Comment (yuanlehome) [inline paddle/phi/infermeta/fusion.cc:1846]: 严格来说，这里也需要设置dtype和layout

**Review by phlrain — APPROVED** [review]
LGTM for check_dygaph =False

**Comment by Tom-Zheng** [comment]
@zyfncg Hi, would you please merge?

File: paddle/phi/api/yaml/fused_ops.yaml (hunk 1 above)
Comment (Tom-Zheng) [reply paddle/phi/api/yaml/fused_ops.yaml:212]: 会在下个PR修改.

File: paddle/phi/infermeta/fusion.cc (hunk 2 above)
Comment (Tom-Zheng) [reply paddle/phi/infermeta/fusion.cc:1846]: 会在下个PR修改.

**Pull request merged**